pytorch_blade/src/torch-mlir/include/torch-mlir/Conversion/TorchToMhlo/MhloLegalizeUtils.h

```diff
@@ -218,6 +218,14 @@ mlir::Value getRollTensor(
     int64_t shiftInt,
     int64_t dim);
 
+Value getBroadcastTensor(
```

Comment:
I note that the implementation of this function have a potential bug for rank-0 case.  we need to explicitly provide the output type when `dimSizes` is empty. Better to change the implementation as well. 
```
Value mhloShape = rewriter.create<tensor::FromElementsOp>(loc, dimSizes);
```